x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/analysis_fields_table.tsx

```diff
@@ -67,141 +69,141 @@ const columns = [
-        },
+    setSortableProperties(sortableProps);
+  }, []);
+
+  const filters = [
+    {
+      type: 'field_value_selection',
+      field: 'is_included',
+      name: i18n.translate('xpack.ml.dataframe.analytics.create.excludedFilterLabel', {
+        defaultMessage: 'Is included',
+      }),
+      multiSelect: false,
+      options: [
         {
-          name: 'is_included',
-          getValue: (item: any) => item.is_included,
-          isAscending: true,
+          value: true,
+          view: (
+            <EuiText grow={false}>
+              {i18n.translate('xpack.ml.dataframe.analytics.create.isIncludedOption', {
+                defaultMessage: 'Yes',
+              })}
+            </EuiText>
+          ),
         },
         {
-          name: 'is_required',
-          getValue: (item: any) => item.is_required,
-          isAscending: true,
+          value: false,
+          view: (
+            <EuiText grow={false}>
+              {i18n.translate('xpack.ml.dataframe.analytics.create.isNotIncludedOption', {
+                defaultMessage: 'No',
+              })}
+            </EuiText>
+          ),
         },
-      ];
-      const sortableProps = new SortableProperties(sortablePropertyItems, defaultSortProperty);
-
-      setSortableProperties(sortableProps);
-    }, []);
+      ],
+    },
+  ];
 
-    const filters = [
-      {
-        type: 'field_value_selection',
-        field: 'is_included',
-        name: i18n.translate('xpack.ml.dataframe.analytics.create.excludedFilterLabel', {
-          defaultMessage: 'Is included',
-        }),
-        multiSelect: false,
-        options: [
-          {
-            value: true,
-            view: (
-              <EuiText grow={false}>
-                {i18n.translate('xpack.ml.dataframe.analytics.create.isIncludedOption', {
-                  defaultMessage: 'Yes',
-                })}
-              </EuiText>
-            ),
-          },
-          {
-            value: false,
-            view: (
-              <EuiText grow={false}>
-                {i18n.translate('xpack.ml.dataframe.analytics.create.isNotIncludedOption', {
-                  defaultMessage: 'No',
-                })}
-              </EuiText>
-            ),
-          },
-        ],
-      },
-    ];
-
-    return (
-      <Fragment>
-        <EuiFormRow
-          label={i18n.translate('xpack.ml.dataframe.analytics.create.excludedFieldsLabel', {
-            defaultMessage: 'Excluded fields',
+  return (
+    <Fragment>
+      <EuiFormRow
+        label={i18n.translate('xpack.ml.dataframe.analytics.create.includedFieldsLabel', {
+          defaultMessage: 'Included fields',
+        })}
+      >
+        <Fragment />
+      </EuiFormRow>
+      {tableItems.length > 0 && (
+        <EuiText size="xs">
+          {includes.length}
+          {i18n.translate('xpack.ml.dataframe.analytics.create.includedFieldsCount', {
+            defaultMessage: ' fields included in the analysis',
```

Comment:
We should switch to `field` in the i18n message if they only have 1 field selected:

![image](https://user-images.githubusercontent.com/7405507/86121183-7ad40a00-bacd-11ea-8386-3afc0d6149f8.png)

Also the error message here needs extra clarification I think - to say that there must be one field in addition to the dependent variable.

Reply:
Updated in 25f859e29c59017dd8b1d4e7085b2a1cd7a3de22